### PR TITLE
Option to disable the mark all as read dialog

### PR DESCRIPTION
**Pull request by GoneUp**
This is a fix for #1923 and my first contribution to this project.
I tested it and it looks like it's working. Do I need to add something else? What happens with the translations for the newly added string resource?

### CODE REVIEW

File: k9mail/src/main/java/com/fsck/k9/preferences/GlobalSettings.java [hunk 1]
```diff
@@ -62,6 +62,9 @@
         s.put("confirmSpam", Settings.versions(
                 new V(1, new BooleanSetting(false))
             ));
+        s.put("confirmMarkAllRead", Settings.versions(
+                new V(44, new BooleanSetting(false))
```
Comment (cketti) [inline k9mail/src/main/java/com/fsck/k9/preferences/GlobalSettings.java]: The default value should be `true`.

**Comment by GoneUp** [comment]
Okay, changed that value and fixed the test that failed at travis.

**Comment by cketti** [comment]
Thanks. I rewrote the previously failing test in PR #1969.
Once the other PR has been merged I'll fix up and merge this one.

**Comment by cketti** [comment]
Cleaned up and merged: https://github.com/k9mail/k-9/compare/580a828b58befc88bcb240fd75a40ac0931a7b50...d0c137662e4572f9b0f2099a2b980ed8a1615a89

**Pull request closed without merging**